Example_of_Game/Practice_Animation/IIT2024245_Kavyan.py

```diff
@@ -0,0 +1,116 @@
+import pygame
+import sys
+
+# Initialize Pygame
+pygame.init()
+
+# Screen dimensions
+SCREEN_WIDTH, SCREEN_HEIGHT = 800, 600
+
+# Colors
+WHITE = (255, 255, 255)
+
+# Initialize screen
+screen = pygame.display.set_mode((SCREEN_WIDTH, SCREEN_HEIGHT))
+pygame.display.set_caption("Sprite Animation Practice")
+
+# Clock for controlling frame rate
+clock = pygame.time.Clock()
+
+# Player class
+class Player:
+    def __init__(self, x, y):
+        self.x = x
+        self.y = y
+        self.speed = 4  # Movement speed
+
+        # Load the main spritesheet
+        self.sprite_sheet = pygame.image.load("SwarmShot/Sprites/Sprites_Pet/PET_Racoon.png").convert_alpha()

```

Comment:
# Load the main spritesheet
        self.sprite_sheet = pygame.image.load("SwarmShot/Sprites/Sprites_Pet/PET_Racoon.png").convert_alpha()
        
   SwarmShot is the directory you already are in , so path shall be : 
    self.sprite_sheet = pygame.image.load("Sprites/Sprites_Pet/PET_Racoon.png").convert_alpha()